epistrains/solver.py

```diff
@@ -189,3 +189,43 @@ def save_compartments(self, save_path='epistrains_output.png'):
 
         plt = self._make_plot()
         plt.savefig(save_path, dpi=300)
+
+    def _make_death_plot(self):
+        """Creates the plot of the number of individuals
+        in each compartment over time
+        """
+        if self.solution is None:
+            raise ValueError("Must run s.solve() before plotting solutions")
+
```

Comment:
I would prefer making the cumulative line dark blue. And maybe also create a parameter for the colours to be consistent with the other plotting function.